components/Banner/src/MDCBannerView.m

```diff
@@ -48,9 +48,9 @@ @interface MDCBannerView ()
 
 // Image constraints
 @property(nonatomic, readwrite, strong) NSLayoutConstraint *imageViewConstraintLeading;
-@property(nonatomic, readwrite, strong) NSLayoutConstraint *imageViewConstraintTopSmall;
+@property(nonatomic, readwrite, strong) NSLayoutConstraint *imageViewConstraintCenterY;
```

Comment:
Please add docs.

Reply:
Followed up in https://github.com/material-components/material-components-ios/issues/7650. Will be done altogether when finalized.